v_3_7_0/master_template.go

```diff
@@ -1585,7 +1585,6 @@ write_files:
         enabled: false # Deafults to true as of 1.10
     authorization:
       mode: AlwaysAllow # Deafults to webhook as of 1.10
-    readOnlyPort: 10255 # Used by heapster. Defaults to 0 (disabled) as of 1.10. Needed for metrics.
```

Comment:
Shouldn't those changes be also ported to `v_4_0_0`?

Reply:
4.0.0 is ignition. I haven't looked into that so far.

Reply:
I’ll port all the changes (which happened after ignition base commit) after operators will be ready. So no worries about that